### ISSUE TITLE
Falcon misses DialogFragments and AlertDialogs

### ISSUE THREAD

**Issue by arkus7** [jraska/Falcon#11]
I'm trying to use Falcon in my app, but it seems to don't save the view of DialogFragments and AlertDialogs. It seems strange, but my app looks nearly same as your sample.

```java

public class MyActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        Toast.makeText(this, "Asd", Toast.LENGTH_LONG).show();
        Snackbar.make(findViewById(android.R.id.content), "snackbar", Snackbar.LENGTH_LONG);
        AlertDialog.Builder builder = new AlertDialog.Builder(this);
        builder.setTitle("Falcon");
        builder.setMessage("Click button below to take screenshot with dialog");
        builder.setPositiveButton("Screenshot", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                try {
                    Falcon.takeScreenshot(this, File.createTempFile("asd", "asd"));
                } catch(IOException e) {
                    e.printStackTrace();
                }
            }
        });
        builder.show();

        // new DialogFragment().show(getSupportFragmentManager(), "DIALOG");
    }

```

Screenshot has view of activity, with Toast and Snackbar, but the dialog is missing. Same for DialogFragment (commented out).

**Comment by jraska** [comment]
Hi, that seems strange, capturing dialogs work and it is the main feature of Falcon. Is my sample working with your device?

**Comment by arkus7** [comment]
Yes, sample is working, but when I copied code from your sample to my app
it wasn't working.

**Comment by jraska** [comment]
Ok so I finally found the problem. When the dialog is shown directly in onCreate method of Activity it is missed in screenshot, because it is overdrawn by activity layout, which is added to global layout manager after the dialog in this case. I will have a look if it is possible to fix it somehow.

Thank you for reporting it.

**Comment by jraska** [comment]
Fix was released now, please give a try version 1.0.3.
